service/models.py

```diff
@@ -18,23 +18,27 @@ class DataValidationError(Exception):
     pass
 
 
-class YourResourceModel(db.Model):
+class Customer(db.Model):
     """
-    Class that represents a <your resource model name>
+    Class that represents a Customer
     """
 
     app = None
 
     # Table Schema
     id = db.Column(db.Integer, primary_key=True)
-    name = db.Column(db.String(63))
+    username = db.Column(db.String(16), unique=True, nullable=False)
```

Comment:
Can the username and password be longer? I would prefer a length of 32 or 64.

I think the following initialization is more consistent to me. Just for your reference :)

username = db.Column(db.String(64), unique=True, nullable=False)
password = db.Column(db.String(64), nullable=False)
first_name = db.Column(db.String(32), nullable=False)
last_name = db.Column(db.String(32), nullable=False)
addresses = db.Column(db.ARRAY(db.String(100)), nullable=False)

Reply:
Ok, sounds good. I've changed the String sizes accordingly